src/app/pages/common/entity/entity-tree-table/entity-tree-table.component.html

```diff
@@ -27,12 +27,25 @@
             <span class="icons">
               <span *ngIf="column.prop === 'name' && element.data.hasOwnProperty('is_encrypted_root') && element.data.hasOwnProperty('non_encrypted_on_encrypted')">
                 &nbsp;
-                <mat-icon role="img" fontSet="mdi-set" fontIcon="mdi-lock-open-variant" *ngIf="element.data.is_encrypted_root && !element.data.locked">
-                </mat-icon>
-                <mat-icon role="img" fontSet="mdi-set" fontIcon="mdi-lock" *ngIf="element.data.is_encrypted_root && element.data.locked">
-                </mat-icon>
-                <mat-icon svgIcon="anti-lock" *ngIf="element.data.non_encrypted_on_encrypted || (!element.data.encrypted && element.data.has_encrypted_children)">
-                </mat-icon>
+                <mat-icon
+                  role="img"
+                  fontSet="mdi-set"
+                  fontIcon="mdi-lock-open-variant" 
+                  *ngIf="element.data.is_encrypted_root && !element.data.locked"
+                  [matTooltip]="'Encrypted dataset is unlocked' | translate"
+                ></mat-icon>
+                <mat-icon 
+                  role="img" 
+                  fontSet="mdi-set" 
+                  [matTooltip]="'Encrypted dataset is locked' | translate"
+                  fontIcon="mdi-lock" 
+                  *ngIf="element.data.is_encrypted_root && element.data.locked"
+                ></mat-icon>
+                <mat-icon 
+                  svgIcon="anti-lock" 
+                  *ngIf="element.data.non_encrypted_on_encrypted || (!element.data.encrypted && element.data.has_encrypted_children)"
+                  [matTooltip]="'Pool has encrypted children' | translate"
```

Comment:
Pool has datasets. Better terminology might be that root dataset has encrypted children. We should also reword this to show that the root dataset is not encrypted itself but has children which are encrypted.